pycript/Reqcheck.py

```diff
@@ -204,40 +191,33 @@ def encrypt_and_update_parameters(extender, currentreq, encryptionpath, selected
                 break
```

Comment:
The same issues are here logic for JSON is broken and will raise an error, and the else print is unwanted and will raise an unwanted error in the "BOTH" as method.